### PR TITLE
Helpfix

### PR DESCRIPTION
**Pull request by ceford**
Pull Request for Issue # .

### Summary of Changes

This pull request adds links to two new Help screens and fixes a problem with the Help button in the Who's online module

### Testing Instructions

1. Go to the **Modules: Privacy Status Check** page and click the Help button in the Toolbar
2. Go to the **Modules: Sample Data** page and click the Help button in the Toolbar
3. Go to the **Modules: Who's Online** page and click the Help button in the Toolbar

### Actual result BEFORE applying this Pull Request

1. 404 Not Found
2. 404 Not found
3. Help button inactive

### Expected result AFTER applying this Pull Request

All deliver working Help pages

### Documentation Changes Required

None

### CODE REVIEW

File: api/language/en-GB/joomla.ini [hunk 1]
```diff
@@ -816,7 +818,7 @@ JHELP_EXTENSIONS_MODULE_MANAGER_STATISTICS="Site_Modules:_Statistics"
 JHELP_EXTENSIONS_MODULE_MANAGER_SYNDICATION_FEEDS="Site_Modules:_Syndication_Feeds"
 JHELP_EXTENSIONS_MODULE_MANAGER_TAGS_POPULAR="Site_Modules:_Tags_-_Popular"
 JHELP_EXTENSIONS_MODULE_MANAGER_TAGS_SIMILAR="Site_Modules:_Tags_-_Similar"
-JHELP_EXTENSIONS_MODULE_MANAGER_WHO_ONLINE="Site_Modules:_Who's_Online"
+JHELP_EXTENSIONS_MODULE_MANAGER_WHO_ONLINE="Site_Modules:_Who%27s_Online"
```
Comment (brianteeman) [inline api/language/en-GB/joomla.ini:821]: Might as well revert this bacck to the J3 version 
"Who_Online"

Simpler and less prone to error

Comment (ceford) [reply api/language/en-GB/joomla.ini:821]: The page title is **Modules: Who's Online**
The Help page title is **Site Modules: Who's Online**
The link used to fetch the Help page is:
https://help.joomla.org/proxy?keyref=Help40:Site_Modules:_Who's_Online&lang=en
But if the apostrophe is used in the translation string it breaks the onclick call of the Help button.
To revert to Who_Online would need the page to be renamed and the Help page title would be **Site Modules: Who Online** which is not good English. Using %27 in place of the apostrophe just works and makes for good English!

**Comment by MartijnMaandag** [comment]
I have tested this item :white_check_mark: successfully on 088a9a2870759db7dc29bed5e60c7f6d80354d9a<br /><br />Yes thanks!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31237">issues.joomla.org/tracker/joomla-cms/31237</a>.</sub>

File: api/language/en-GB/joomla.ini (hunk 1 above)
Comment (brianteeman) [reply api/language/en-GB/joomla.ini:821]: good english != url

**Comment by particthistle** [comment]
I have tested this item :white_check_mark: successfully on 088a9a2870759db7dc29bed5e60c7f6d80354d9a<br /><br />Nice work again @ceford 
Tested successfully. All three help screens now appear correctly.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31237">issues.joomla.org/tracker/joomla-cms/31237</a>.</sub>

**Comment by particthistle** [comment]
Watching for when retest is required after @ceford applies change suggested by @brianteeman

**Comment by ceford** [comment]
Unless @brianteeman, or someone else, can provide a good reason not to use %27 in place of an apostrophe in "Site_Modules:_Who%27s_Online" I propose to leave it as it is. This string is actually a 'not to be translated' url (marked as such in Crowdin). This is what the top of the Help screen looks like in Joomla 4:

>image

And in Joomla 3:

>image

I don't see the point of reverting to the Joomla 3 way.

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31237">issues.joomla.org/tracker/joomla-cms/31237</a>.</sub>

**Pull request merged**